src/components/compound/DlTreeTable/views/DlTrTreeView.vue

```diff
@@ -53,6 +53,7 @@
                     }px;`
             "
             :col-index="colIndex"
+            :tool-tip="toolTip"
```

Comment:
When changing tooltip to lowercase, the prop should be one work `:tooltip="tooltip"`